content/scalability/docs/supporting-infrastructure.md

```diff
@@ -0,0 +1,134 @@
+
+Global Accelerator can route workloads to the nearest region based on request IP address. This may be useful for workloads that are deployed to multiple regions, but it does not improve routing to a single cluster in a single region. Using Route 53 in combination with the Global Accelerator has additional benefits such as health checking and automatic failover if an AZ is not available. You can see an example of using Global Accelerator with Route 53 in [this blog post](https://aws.amazon.com/blogs/containers/operating-a-multi-regional-stateless-application-using-amazon-eks/).
+
+CloudFront can be use with Route 53 and Global Accelerator or by itself to route traffic to multiple destinations. CloudFront caches assets being served from the origin sources which may reduce bandwidth requirements depending on what you are serving.
+
+## Use AWS Systems Manager Patch Manager to automate patching nodes
+
+To patch a Linux host it takes seconds to install a package which can be run in parallel without disrupting containerized workloads. The package can be installed and validated without cordoning, draining, or replacing the instance.
+
+To replace an instance you first need to create, validate, and distribute new AMIs. The instance needs to have a replacement created, and the old instance needs to be cordoned and drained. Then workloads need to be created on the new instance, verified, and repeated for all instances that need to be patched. It takes hours, days, or weeks to replace all instances in a large cluster safely without disrupting workloads.
+
+Amazon recommends using immutable infrastructure that is built, tested, and promoted from an automated, declarative system, but if you have clusters larger than 300 nodes and you have a requirement to patch systems in 24 hours or less then you will likely need to patch existing systems in place. Because of the large time differential between patching and replacing systems we recommend using [AWS Systems Manager Patch Manager](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-patch.html) to automate patching nodes when required to do so. This will allow you to quickly roll out security updates and replace the instances on a regular schedule after your base AMI has been patched. If you are using Bottlerocket we recommend using the [Bottlerocket update operator](https://github.com/bottlerocket-os/bottlerocket-update-operator) to keep your nodes up to date.
```

Comment:
"clusters larger than 300 nodes" - can you add couple of lines to explain the significance of the "300" number

Reply:
There is no significance to 300. It's just a common number I got from support and engineering.

Reply:
got it. Maybe just adding a sentence saying something to the extend "From our experience supporting various customers on EKS, we have seen that 300...."